packages/medusa/src/loaders/__tests__/models.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { asValue } from "awilix"
+import { createMedusaContainer } from "medusa-core-utils"
+import path, { resolve } from "path"
+import { DataSource, EntityManager } from "typeorm"
+
+import Logger from "../logger"
+import { MEDUSA_PROJECT_NAME, registerServices, registerStrategies } from "../plugins"
+import modelsLoader from "../models"
+
+describe("models loader", () => {
```

Comment:
@olivermrbl @adrien2p added some tests. This was a bit of an interesting one to do.